src/test/counters.test.js

```diff
@@ -0,0 +1,37 @@
+const { countItems } = require('../modules/counters.js');
+
+describe('Counting number of movies on the page', () => {
+  test('test number of items', () => {
```

Comment:
- [ ] Kindly update the test for the `count-items` function based on the new implementation and please make sure that your test includes all edge cases like if there are 0 items on the page.